README.md

```diff
@@ -0,0 +1,33 @@
+# Relectric Systems
```

Comment:
I'll probably clean up the README later once I get more actions in but this is good for now